examples/ex04/ex04b/2BlockFriction_Isoilunit.i

```diff
@@ -0,0 +1,486 @@
+[Mesh]
+    type = FileMesh
+    file = '2blockfriction_isoilunit.e'
+[]
+
+[GlobalParams]
+    displacements = 'disp_x disp_y disp_z'
+[]
+
+[Variables]
+    [./disp_x]
+    [../]
+    [./disp_y]
+    [../]
+    [./disp_z]
+    [../]
+[]
+
+[AuxVariables]
+    [./stress_xy]
+      order = CONSTANT
+      family = MONOMIAL
+    [../]
+    [./stress_xx]
+      order = CONSTANT
+      family = MONOMIAL
+    [../]
+    [./stress_yy]
+      order = CONSTANT
+      family = MONOMIAL
+    [../]
+    [./stress_zz]
+      order = CONSTANT
+      family = MONOMIAL
+    [../]
+    [./stress_yz]
+      order = CONSTANT
+      family = MONOMIAL
+    [../]
+    [./stress_zx]
+      order = CONSTANT
+      family = MONOMIAL
+    [../]
+    [./strain_yz]
+      order = CONSTANT
+      family = MONOMIAL
+    [../]
+    [./strain_zx]
+        order = CONSTANT
+        family = MONOMIAL
+    [../]
+    [./strain_xy]
+        order = CONSTANT
+        family = MONOMIAL
+    [../]
+    [./strain_xx]
+        order = CONSTANT
+        family = MONOMIAL
+    [../]
+    [./strain_yy]
+        order = CONSTANT
+        family = MONOMIAL
+    [../]
+    [./strain_zz]
+        order = CONSTANT
+        family = MONOMIAL
+    [../]
+    [./layer_id]
+        order = CONSTANT
+        family = MONOMIAL
+    [../]
+    [./vel_x]
+    [../]
+    [./accel_x]
+    [../]
+    [./vel_y]
+    [../]
+    [./accel_y]
+    [../]
+    [./vel_z]
+    [../]
+    [./accel_z]
+    [../]
+[]
+
+[Kernels]
+    [./DynamicTensorMechanics]
+      displacements = 'disp_x disp_y disp_z'
+      zeta = 0.00006366
+      [../]
+      [./gravity]
+      type = Gravity
+      variable = disp_z
+      value = -9.81
+    [../]
+    [./inertia_x]
+      type = InertialForce
+      variable = disp_x
+      velocity = 'vel_x'
+      acceleration = 'accel_x'
+      beta = 0.25
+      gamma = 0.5
+      eta = 0
+    [../]
+    [./inertia_y]
+      type = InertialForce
+      variable = disp_y
+      velocity = vel_y
+      acceleration = accel_y
+      beta = 0.25
+      gamma = 0.5
+      eta = 0
+    [../]
+    [./inertia_z]
+      type = InertialForce
+      variable = disp_z
+      velocity = vel_z
+      acceleration = accel_z
+      beta = 0.25
+      gamma = 0.5
+      eta = 0
+    [../]
```

Comment:
Just wanted to check if you tried re-running the problem with `use_displaced_mesh = true` in inertia and gravity kernels.